thirdparty/djvulibre/CMakeLists.txt

```diff
@@ -32,6 +32,8 @@ if($ENV{ANDROID})
     set(CFG_CMD "${CFG_CMD} && ${ISED} 's|soname_spec=.*|soname_spec=\"\\\\\$libname\\\\\$release\\\\\$shared_ext\\\\\$major\"|' libtool")
 endif()
 
+# Apparently CMake is scoped.
```

Comment:
I don't think what I wrote late last night makes sense given that `else` is just as scoped as `if`. :-P

I don't get it though; $ENV{ANDROID} should always combine with NOT $ENV{WIN32}.